### ISSUE TITLE
The search with ENTER key performed with wrong phrase if mouse pointer is over suggestion list.

### ISSUE THREAD

**Issue by alice0775** [mozilla/addons-frontend#7655]
Describe the problem and steps to reproduce it:
Search performed with unexpected words if the mouse pointer is below the search field(In other words if mouse pointer is over suggestion list.).

(Please include as many details as possible.)
1.  type "uBlock Origin" in the search input field
 2. hit ENTER

What happened?
A non-related addon page displays

What did you expect to happen?
Search should be performed with inputed term.

Anything else we should know?

**Comment by suhailsinghbains** [comment]
Hi @alice0775 
Can you please attach a screenshot or preferably, a GIF as I am not able to reproduce this issue.
Thanks

**Comment by alice0775** [comment]
> 
> 
> Hi @alice0775
> Can you please attach a screenshot or preferably, a GIF as I am not able to reproduce this issue.
> Thanks

Before hit ENTERT key, It needs to move the mouse pointer over the suggestions list(not intentionally, but due to vibration of desk or your arm hits the mouse etc... For various reasons , the mouse pointer can move)

See screencast, https://www.youtube.com/watch?v=lzgxx8GcO1s

**Comment by suhailsinghbains** [comment]
I would like to take this up.
I think we have to check the keycode of the last input char, [here](https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/AutoSearchInput/index.js#L237-L238)
But, as soon I press enter, this code doesn't work at all.
I would need help here

**Comment by muffinresearch** [comment]
@suhailsinghbains thanks for taking a look.

@willdurand would you be able to provide some help to @suhailsinghbains on this?

**Comment by mirefly** [comment]
It looks like intended behavior of `react-autosuggest` (selecting the highlighted suggestion when pressing Enter).

**Comment by willdurand** [comment]
As @mirefly pointed out, it looks like the intended behavior. Maybe we could prevent hover+select with the mouse, but I am not sure.

**Comment by willdurand** [comment]
I believe this issue in the react-autosuggest repo is the cause of this bug: https://github.com/moroshko/react-autosuggest/issues/378.

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDFRNT-163
